index.js

```diff
@@ -1,17 +1,73 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
 
-  add(item) {}
+  add(item) {
+    let isInserted = false;
+    for (let i = 0; i < this.length; i++) {
+      if (item <= this.items[i]) {
+        this.items.splice(i, 1, item, this.items[i]);
+        isInserted = true;
+        break;
+      }
+    }
+    if (isInserted === false) {
+      this.items.push(item);
+    }
+    this.length++;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.length) {
+      throw new Error("OutofBounds");
+    } else {
```

Comment:
Quando vc damos um throw Error, o código automaticamente para de executar, então vc não precisa colocar o else ;-)

Caso não caia no IF somente retorne o elemento

Outro ponto: no seu IF, vc pode ter um problema caso o pos seja negativo. Vc pode fazer uma verificação usando o próprio this.items[pos]

if (!this.items[pos]) {
    throw new Error("OutofBounds");
}

return this.items[pos];